.github/workflows/deploy.yml

```
@@ -0,0 +1,80 @@
+        with:
+          host: ${{ secrets.EC2_HOST }}
+          username: ${{ secrets.EC2_USERNAME }}
+          key: ${{ secrets.EC2_SSH_PRIVATE_KEY }}
+          script: |
+            # 進入 EC2 上的部署目錄
+            cd /home/ubuntu/user-service
+
+            # 將 CI/CD 流程中的變數寫入環境變數檔案
+            # 這一行會建立 .env 檔案，裡面包含了最新映像檔的完整 URI，給 docker-compose.yml 使用
+            echo "IMAGE_TAG=${{ steps.build-image.outputs.image_uri }}" > .env
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)